demo/stages-pod-general.demo

```diff
@@ -1,5 +1,5 @@
 # Let's getting started with kwokctl!
-kwokctl -c ./stages/pod-general.yaml create cluster
+kwokctl -c ./kustomize/stage/pod/general/kustomization.yaml create cluster
```

Comment:
Ah, now is unsupported use kustomization.yaml file as config, it will support in #1018

Reply:
Any fix without the `kcustomization` support?